benchmark/reformat_input.py

```diff
@@ -113,8 +113,8 @@ def species_gene_mapping(gene_tree,
         else:
             raise ValueError(
                 "Species %s does not exist in the species tree" % species)
-
-    return mapping_leaves
+    return OrderedDict(sorted(mapping_leaves.items(),
```

Comment:
the doc string should be OrderedDict, will update. The reason for OrderedDict is so that when I output the dictionary to a file the entries are always in the same order, easier for debugging and testing ..